### PR TITLE
💄962 - Make link underline red

### PR DESCRIPTION
**Pull request by Aibono1225**
- [x] Recommended extensions have been installed if using VS Code

This is to solve https://github.com/SSWConsulting/SSW.Rules/issues/962

Changes I made:
- Added styles that make the link underline red on hover

image
image

### CODE REVIEW

File: src/components/floating-bubble/floating-bubble.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <button
+      onClick={() => handleBubbleClick()}
+      className="fixed bottom-4 right-4 w-16 h-16 bg-ssw-red rounded-full shadow-lg flex items-center justify-center cursor-pointer"
+    >
+      <FontAwesomeIcon icon={faComments} size="2x" className="text-white" />
+    </button>
+  );
+};
+
+export default FloatingBubble;
```
Comment (BrookJeynes) [inline src/components/floating-bubble/floating-bubble.js:20]: Should this file be included in this PR?

File: src/components/layout/layout.js [hunk 2]
```diff
@@ -66,6 +67,7 @@ const Layout = ({ children, displayActions, ruleUri, crumbLabel }) => {
           <Footer />
         </div>
 
+        <FloatingBubble />
         <MobileMenu isMenuOpened={isMenuOpened}></MobileMenu>
       </Auth0Provider>
     </div>
```
Comment (BrookJeynes) [inline src/components/layout/layout.js:73]: Should this change be included in this PR?

**Review by BrookJeynes — CHANGES_REQUESTED** [review]
Style changes look good, just remove the other changes from this PR :)

File: src/components/floating-bubble/floating-bubble.js (hunk 1 above)
Comment (Aibono1225) [reply src/components/floating-bubble/floating-bubble.js]: Done - I deleted incorrect commit and code for this :)

File: src/components/layout/layout.js (hunk 2 above)
Comment (Aibono1225) [reply src/components/layout/layout.js]: Done - this change shouldn't be here and I removed it as well

**Review by BrookJeynes — APPROVED** [review]
LGTM

**Pull request merged**